### PR TITLE
ap20/ap21 migration

### PR DESCRIPTION
**Pull request by rynge**
Not complete, but wanted this PR here for a discussion on how we should document the migration.

### CODE REVIEW

**Comment by rynge** [comment]
I changed my mind - let's get this merged sooner rather than later, and then we can iterate. Please review.

File: documentation/overview/account_setup/connect-access.md [hunk 1]
```diff
@@ -60,10 +60,34 @@ be added to the login nodes within a couple hours.
 > a keypair on each computer you want to use, and then adding the public key to your OSG 
 > Connect profile. 
 
+## Add multi factory authentication to your web profile
+
+Multi factory authentication means that you will use 2 different methods to authenticate
+when you log in. The first factor is the ssh key you added above. The second factory
```
Comment (ChristinaLK) [inline documentation/overview/account_setup/connect-access.md:66]:
```suggestion
when you log in. The first factor is the ssh key you added above. The second factor
```

File: documentation/overview/account_setup/connect-access.md [hunk 2]
```diff
@@ -60,10 +60,34 @@ be added to the login nodes within a couple hours.
...
+
+4. Check the "Set up Multi-Factor Authentication" at the bottom and hit Apply.
+
+5. In the Multi-Factory Authentication box, follow the instructions (scan the QR code with your TOTP client)
+
 ## Logging In
 
-After following the steps above to upload your key and it's been about fifteen minutes, you should 
-be able to log in to OSG Connect. 
+After following the steps above to upload your key, set up multi factory authentication, and it's
```
Comment (ChristinaLK) [inline documentation/overview/account_setup/connect-access.md:89]:
```suggestion
After following the steps above to upload your key, set up multi factor authentication, and it has
```

**Review by ChristinaLK — APPROVED** [review]
lots of factory -> factor. ;)

**Comment by ChristinaLK** [comment]
@rynge we should also add something about default environment variables?

**Pull request merged**